tensorflow_lite_support/ios/task/vision/apis/TFLImageClassifier.h

```diff
@@ -0,0 +1,118 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ ==============================================================================*/
+#import <Foundation/Foundation.h>
+
+#import "tensorflow_lite_support/odml/ios/image/apis/GMLImage.h"
+#import "tensorflow_lite_support/ios/task/core/apis/TFLBaseOptions.h"
+#import "tensorflow_lite_support/ios/task/processor/apis/TFLClassificationOptions.h"
+#import "tensorflow_lite_support/ios/task/processor/apis/TFLClassificationResult.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Options to configure TFLImageClassifier.
+ */
+@interface TFLImageClassifierOptions : NSObject
```

Comment:
Each task will also have their own framework so that Google internal users can depend on. Some users are very sensitive about the binary size. But I think we can put all vision apis and source files into the same directory, just like C++.

Reply:
SGTM. 
@priankakariatyml 
Please move the `TFLImageClassifier.h` into `sources` folder together with the implementation file.

@lu-wang-g Internal users will depend on Bazel targets directly right? So as long as we split the target for ImageClassifier vs. ObjectDetector vs. ImageSegmenter, the way how we package external framework (e.g. having all Vision APIs as one framework) doesn't affect internal users.

Reply:
Correct, internal users only care about the blaze targets; where the file is located is less important.

Reply:
Moved all to sources